ci-operator/jobs/openshift/cincinnati-graph-data/openshift-cincinnati-graph-data-master-presubmits.yaml

```diff
@@ -1,5 +1,96 @@
 presubmits:
   openshift/cincinnati-graph-data:
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - master
+    cluster: build01
+    context: ci/prow/e2e
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-cincinnati-graph-data-master-e2e
+    optional: true
```

Comment:
I'm curious if you had to add this line manually, because I can't find an indicator for it in the config

Reply:
Right, this needs to be added manually (along with `always_run: false`) - ci-operator schema doesn't have support for optional/ad-hoc tests yet